imapclient/client.go

```diff
@@ -166,6 +174,9 @@ func New(conn net.Conn, options *Options) *Client {
 	if options == nil {
 		options = &Options{}
 	}
+	if options.CommandsQueueSize <= 0 {
```

Comment:
Remove this if this is acceptable: by default, commands order is not guaranteed.